### PR TITLE
Feat/aylien api parent

### PR DESCRIPTION
**Pull request by aseiden**
Overview:

.gitignore: Just a bit of housekeeping that came up while working on this branch.  Prevents .vscode customization files from sneaking into our repo.

package.json: Date formatting utility

aylienAPI: The main show.  Just noticed a lack of a newline, will add that in right after I submit the PR.

aylienApiDataFormatter: Added in conditional logic to handle missing media data.

### CODE REVIEW

File: utilities/aylienApi.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+const AylienNewsApi = require('aylien-news-api');
+const epoch = require('epoch.js');
```
Comment (epan) [inline utilities/aylienApi.js:2]: FYI you don't need to put extensions in require if it's `.js` I think

File: utilities/aylienApi.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+const formatStories = require('./aylienApiDataFormatter');
+
+const getStories = (peaks, queryString, callback) => {
```
Comment (epan) [inline utilities/aylienApi.js]: Not sure if intentionally deviating from the interface here of 

`getNews(searchString, peaks, (peakStories) => {` 

See: https://github.com/TrendGame/TrendGame/blob/master/utilities/makeTimeline.js#L10

File: utilities/aylienApi.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    appKey.apiKey = apiInfo.key;
+  }
+
+  // Assign options for API query
+  const opts = {
+    'title': queryString,
+    'sortBy': 'source.links_in_count',
+    'language': ['en'],
+    'publishedAtStart': formattedPeakDate,
+    'publishedAtEnd': 'NOW',
```
Comment (epan) [inline utilities/aylienApi.js]: Does this get stories between the peak data and current computer time? Could be 1 year?

File: utilities/aylienApi.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    if (error) {
+      callback(error, null);
+    } else {
+      // 4 chosen so we display 4 stories under the graph.  Not ideal to have this hard coded here, but MVP!
+      // Ideally we only query for 4, but I haven't been able to find a limit query parameter.
+      data.stories.length = 4;
+      const formattedStories = formatStories(data);
+      let finalData = [];
+      finalData.push({
+        date: peaks[0][0],
```
Comment (epan) [inline utilities/aylienApi.js]: Could you reference `peakDate` here?

**Review by epan — COMMENTED** [review]
Most important question is about changing the "getNews" interface

File: utilities/aylienApi.js (hunk 3 above)
Comment (aseiden) [reply utilities/aylienApi.js]: Woops!  No this should be refactored to look for one week after the peak date.  Good catch!

File: utilities/aylienApi.js (hunk 4 above)
Comment (aseiden) [reply utilities/aylienApi.js]: Yes, good catch!

File: utilities/aylienApi.js (hunk 2 above)
Comment (aseiden) [reply utilities/aylienApi.js]: Good catch!

File: utilities/aylienApi.js (hunk 1 above)
Comment (aseiden) [reply utilities/aylienApi.js:2]: Roger roger!

Comment (aseiden) [reply utilities/aylienApi.js:2]: Actually .js is in the title of this package, it needs to stay. :\

Comment (epan) [reply utilities/aylienApi.js:2]: lol epoch.js

**Review by epan — APPROVED** [review]
so hype i fell out my chair 🍸

**Pull request merged**